sdk/identity/identity/src/client/identityClient.ts

```diff
@@ -41,6 +40,22 @@ export interface TokenResponse {
   refreshToken?: string;
 }
 
+/**
+ * @internal
+ */
+export function getIdentityClientAuthorityHost(options?: TokenCredentialOptions): string {
+  // The authorityHost can come from options or from the AZURE_AUTHORITY_HOST environment variable.
+  let authorityHost = options?.authorityHost;
+
+  // The AZURE_AUTHORITY_HOST environment variable can only be provided in NodeJS.
+  if (isNode) {
+    authorityHost = authorityHost || process.env.AZURE_AUTHORITY_HOST;
```

Comment:
```suggestion
    authorityHost = authorityHost ?? process.env.AZURE_AUTHORITY_HOST;
```

I always get sketched out by `||` with string values, because empty string is falsy, especially now that we have this nice nullish coalescing operator.

Reply:
Thank you!